src/WikiService.Data/ArticleRepository.cs

```diff
@@ -34,5 +36,24 @@ public async Task<bool> DoesSameNameExistAsync(Guid rubricId, string articleName
 
       return dbArticle.Id;
     }
+
+    public async Task<DbArticle> GetAsync(GetArticleRequest request)
+    {
+      if (request is null)
+      {
+        return null;
+      }
+
+      DbArticle dbArticle = await _provider.Articles
+        .FirstOrDefaultAsync(article => article.Id == request.ArticleId);
+
+      if (dbArticle is not null) 
+      {
+        dbArticle.Files = await _provider.ArticlesFiles
+          .Where(f => f.ArticleId == dbArticle.Id).ToListAsync();
+      }
+
+      return dbArticle;
```

Comment:
return  await _provider.Articles.Include(a => a.Files).FirstOrDefaultAsync(article => article.Id == request.ArticleId);
and remove from 47 to 55 line